modules/browsiRtdProvider.js

```diff
@@ -184,25 +178,24 @@ export function isIdMatchingAdUnit(id, slot, whitelist) {
 
 /**
  * get GPT slot by placement id
- * @param {string} id placement id
+ * @param {string} code placement id
  * @return {?Object}
  */
-function getSlotById(id) {
+function getSlotByCode(code) {
   const slots = getAllSlots();
   if (!slots || !slots.length) {
     return null;
   }
-  return slots.filter(s => s.getSlotElementId() === id)[0] || null;
+  return slots.find(s => s.getSlotElementId() === code || s.getAdUnitPath() === code) || null;
```

Comment:
`.find` is not supported by ie11, can you please use the find from the core-js lib?

Example Import: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L7

Example Run: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L69

Reply:
fixed